.github/scripts/notifyWhenReadyForDev.js

```diff
@@ -56,6 +56,20 @@ module.exports = async ({ github, context }) => {
       milestone: null,
     });
 
+    // Remove "needs triage" label
+    try {
```

Comment:
Could you use the [`removeLabel`](https://github.com/Esri/calcite-design-system/blob/14f8cff1b5ae6156715171ef4e6d261080cebc31/.github/scripts/support/utils.js#L11) util for this (see [example](https://github.com/Esri/calcite-design-system/blob/1b22e70ec1662138c17d1a9e4c6a79d87794c4ef/.github/scripts/assignForVerification.js#L28))?

Reply:
I just noticed there's a slight difference in behavior compared to the util, so we can also update the util for consistent error handling. cc @benelan